bears/ruby/RuboCopBear.py

```diff
@@ -60,9 +62,9 @@ def generate_config(filename, file,
                                                  "REVIEW"),
                         min_if_unless_guard: int=1,
                         indent_size: int=2,
-                        method_name_case: str='snake_case',
+                        method_naming_convention: str='snake_case',
```

Comment:
agreed, this should be 'snake' instead, not 'snake_case' :p

if someone asks you: what case would you like to use? 'snake'! not 'snake_case'! :p